docs/sources/setup-grafana/configure-security/secret-scan.md

```diff
@@ -0,0 +1,88 @@
+---
+description: Detect and revoke leaked Grafana service account tokens
+labels:
+  products:
+    - cloud
+    - enterprise
+    - oss
+title: Configure Grafana secret scanning
+menuTitle: Configure secret scanning
+weight: 1000
+---
+
+# Configure Grafana secret scanning
```

Comment:
```suggestion
# Configure Grafana secret scanning and notifications
```

Reply:
Added 'and notifications' here to address that this topic contains two tasks. This is just fyi so you understand why I am making this recommendation.